snapcraft_legacy/internal/lifecycle/_init.py

```diff
@@ -42,7 +43,33 @@
 )  # noqa, lines too long.
 
 
-def init():
+def init_git():
+    """Initialize a github repository."""
+    gitignore_path = ".gitignore"
+    _GITIGNORE = dedent(
+        """\
+        *.snap
+        parts/
+        stage/
+        prime/
+        """
+    )
+
+    try:
+        subprocess.check_output(["git", "init"])
+    except subprocess.CalledProcessError as e:
+        raise errors.GitCommandError(
+            command=command,
```

Comment:
This is the line [ruff's complaining about](https://github.com/snapcore/snapcraft/actions/runs/3960247814/jobs/6784632193#step:5:129). I think you intend to set `command = ["git", "init"]` before the try?

Reply:
Ah, yep. Just changed it and rebased!